andela/ah-titans-frontend
# #160011179 User view and edit profile

#### What does this PR do?
A user can view profiles and update his/her profile.

#### Description of tasks to be completed
Implements user view own and other users profile, update own profile and update profile image.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ git checkout ft-user-view-edit-profile-160011179
```
```sh
$ cd ah-titans-frontend
```

> Install the dependencies with the command below.

```sh
$ npm install
```
>Then run the app with the command below.
```sh
$ npm start
```
> You should get to the home page (`http://localhost:4000/profile/:username`)

### Relevant screenshots
> View profile
<img width="1280" alt="screen shot 2018-09-20 at 00 09 29" src="https://user-images.githubusercontent.com/12511966/45781747-9314c500-bc69-11e8-8160-29ab06da7adc.png">


> Update profile
<img width="1280" alt="screen shot 2018-09-20 at 00 09 46" src="https://user-images.githubusercontent.com/12511966/45781755-9c9e2d00-bc69-11e8-982d-ec6e100eb8c3.png">

> Update profile picture
<img width="1280" alt="screen shot 2018-09-20 at 00 09 57" src="https://user-images.githubusercontent.com/12511966/45781790-bb9cbf00-bc69-11e8-8d19-0645981fd1ab.png">


#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160011179](https://www.pivotaltracker.com/story/show/160011179)
